wikipedia-sentences-query-while-indexing/app.py

```diff
@@ -0,0 +1,219 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+import shutil
+import time
+import traceback
+from contextlib import ExitStack
+from pathlib import Path
+from typing import List, Dict
+
+import click
+import requests
+from jina import Document
+from jina.clients.sugary_io import _input_lines
+from jina.logging import JinaLogger
+
+logger = JinaLogger('jina')
+
+curdir = os.getcwd()
+
+JINAD_HOST = 'localhost'  # change this if you are using remote jinad
+JINAD_PORT = '8000'  # change this if you set a different port
+DUMP_PATH = '/tmp/jina_dump'  # the path where to dump
+SHARDS = 3  # change this if you change pods/query_indexer.yml
+DUMP_RELOAD_INTERVAL = 20  # time between dump - rolling update calls
+DATA_FILE = 'data/toy.txt'  # change this if you get the full data
+DOCS_PER_ROUND = 5  # nr of documents to index in each round
+DBMS_REST_PORT = '9000'  # REST port of DBMS Flow, defined in flows/dbms.yml
+QUERY_REST_PORT = '9001'  # REST port of Query Flow, defined in flows/query.yml
+
+
+def query_restful():
+    while True:
```

Comment:
query restful shpuld not wait for the user to send queries? how will this work from docker? can docker capture user's std input?

Reply:
Talked to Fionn, won't be building a docker for this example

Reply:
then remove the Dockerfile, but anyway, if u do query restful I would expect people to send rest queries and not typing right?

Reply:
yes, it's removed and the README includes instructions on how to do REST queries with curl etc.